packages/shared/components/Error.svelte

```diff
@@ -1,10 +1,8 @@
 <script lang="ts">
-    import { Text, TextType } from 'shared/components'
-    export let error = ''
+    import { Text } from 'shared/components'
+    export let error: string
 </script>
 
-{#if error !== null}
```

Comment:
the error wrapper div has a parasite top margin so i dont think we should remove this check here